kubeflow/seldon/json/README.md

```diff
@@ -0,0 +1,7 @@
+The template.json is generated from the helm source chart at https://github.com/SeldonIO/seldon-core/tree/master/helm-charts/seldon-core
+
+```
+helm template ${SELDON-CORE-HOME}/helm-charts/seldon-core > template.yaml
```

Comment:
Did you forget to add the .json files? Based on the diff it looks like a lot of .json files are deleted and I don't see new ones.

Reply:
All the json is now in template.json which is generated from our centralized helm chart using ```helm template```